scripts/stop-server

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import logging
+import os
+import pwd
+import subprocess
+import sys
+import time
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from scripts.lib.zulip_tools import (
+    ENDC,
+    OKGREEN,
+    WARNING,
+)
+
+if pwd.getpwuid(os.getuid()).pw_name != "zulip":
+    logging.error("Must be run as user 'zulip'.")
+    sys.exit(1)
+
+logging.Formatter.converter = time.gmtime
+logging.basicConfig(format="%(asctime)s stop-server: %(message)s", level=logging.INFO)
+
+def stop(label: str, *processes: str) -> None:
+    logging.info(f"Stopping {label}")
+    subprocess.check_call(["supervisorctl", "stop", *processes])
+
+stop("Tornado", "zulip-tornado", "zulip-tornado:*")
+stop("workers", "zulip-workers:*")
+stop("server core", "zulip-django")
+
+if os.path.exists("/etc/supervisor/conf.d/zulip/thumbor.conf"):
+    stop("thumbor thumbnailing service", "zulip-thumbor")
+
+if os.path.exists("/etc/supervisor/conf.d/zulip/zulip_db.conf"):
+    stop("full-text indexing", "process-fts-updates")
+
+print()
+print(OKGREEN + "Application stopped successfully!" + ENDC)
```

Comment:
Should this say "Zulip stopped successfully"?

Reply:
Sure.  We say "Application restarted successfully" in `restart-server`.